language/documentation/book/src/bool.md

```diff
@@ -16,7 +16,7 @@ Literals for `bool` are either `true` or `false`.
 | Syntax   | Description | Equivalent Expression |
 | -------- | ----------- | --------------------- |
 | `&&` | short-circuiting logical and | `p && q` is equivalent to `if (p) q else false` |
-| <code>&#124;</code> | short-circuiting logical or |`p <code>&#124;</code> q` is equivalent to `if (p) true else q` |
+| `||` | short-circuiting logical or |`p || q` is equivalent to `if (p) true else q` |
```

Comment:
Hopefully this renders? There were some issues in the past of different markdown renderers not being happy with it

Reply:
I just checked in the mdbook in `main`and it doesn't :(. I think maybe this `<code>` thing was specific to hackmd?

Reply:
Apparently it does not, I'll see how to work around this as soon as I know how to preview this!

Reply:
After trying it here https://mdxjs.com/playground I think I found a version using `<code>` that renders properly. Just pushed it.
Markdown should really have been standardized properly :sweat_smile: